projects.yaml

```diff
@@ -1928,3 +1935,9 @@ calculated from density functional theory as the objective function."
   license: GPL-3.0
   github_id: jeherr/element-encoder
   # package managers: none
+- name: MODNet
+  category: general-tool
```

Comment:
Will change to `category: rep-eng` ([Representation engineering](https://github.com/JuDFTteam/best-of-atomistic-machine-learning#representation-engineering)), aka feature engineering. Reason: I would argue that MODNet falls under feature engineering, since the initial compositional and/or structural feature set input to the network are hand-picked. Quote from [original publication](https://doi.org/10.1038/s41524-021-00552-2).

> Depending on the amount of available data, a clear distinction should thus be made between feature- [eg MODNet, `rep-eng`] and graph-based models [eg MEGNet, `rep-learn`]. The former should be preferred for small to medium datasets, while the latter should be left for large datasets, as it will be confirmed for the vibrational properties.